### PR TITLE
[Card][Clickable] Add focus style

### PR DESCRIPTION
**Pull request by jeremie-lucca**
## Description

Add focus style to card clickable

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: https://lucca-front.lucca.tech/PR-2392/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://qa.lucca.tech/8290/ui-diff)

File: stories/qa/card/card.stories.html [hunk 1]
```diff
@@ -59,7 +59,7 @@ <h2 class="card-title">Title</h2>
 <!-- States -->
 <section class="contentSection">
 	<h2 class="card-title">States</h2>
-	<a class="card mod-clickable">
+	<a href class="card mod-clickable">
```
Comment (vvalentin-lucca) [inline stories/qa/card/card.stories.html:62]: We need to rethink this to avoid the wrap-around link.

Comment (jeremie-lucca) [reply stories/qa/card/card.stories.html:62]: Agreed, but it's just a fix on a deprecated mod. I guess we are going to improve `mod-action` in the future.

**Review by AlexiaAlarza — APPROVED** [review]

**Pull request merged**